src/server/models/match.model.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+import mongoose from 'mongoose';
+const Schema = mongoose.Schema;
+
+let matchSchema = new Schema({
+  mentorId: Schema.Types.ObjectId,
```

Comment:
because this is an ObjectId, email address should be called from DB.
But I didn't connect DB yet so hard coded in match.controller.js
I'll add TODO